services/user-svc/api/user-svc.proto

```diff
@@ -18,6 +18,21 @@ message CreateUserResponse {
 	string userID = 1;
 }
 
+//
+// Update User
+//
+
+message UpdateUserRequest {
+	// if userId is missing the requesting user's id is used
+	optional string userId = 1; // @gotags: validate:"omitempty,uuid4"
```

Comment:
This should be required.

Reply:
Why?

Reply:
Sorry, my bad. Didn't catch the `hwutil.DerefStringOr`.